Inline Help: Adjust footer to allow longer labels

evilluendas opened the pull request:
#### Changes proposed in this Pull Request

* Adjust footer in Inline Help so the layout is vertical instead of horizontal.
* Add padding to the right of the "Contact us" button so the chevron never overlaps.

The footer in the Inline Help popover broke line very weirdly when the label was too long, which could happen depending on the language selected for the UI. This PR adjusts the layout so it's vertical, making it more scalable (labels can be much longer now) and more consistent (it looks the same regardless of the language selected).

Before: 
![image](https://user-images.githubusercontent.com/820374/59388294-45247d80-8d20-11e9-81c1-24ff37dbaa7f.png)


After:
![image](https://user-images.githubusercontent.com/820374/59388250-28884580-8d20-11e9-8366-0455cd08a655.png)

#### Testing instructions

* Go to Calypso.
* Open the Inline Help by clicking on the `?` icon on the lower-right corner.
* Make sure it looks fine regardless of your UI language.

Fixes #33460

sixhours commented:
With this change, languages that have less text look broken/off balance:

<img width="352" alt="Screen Shot 2019-06-12 at 3 00 12 PM" src="https://user-images.githubusercontent.com/2124984/59389282-ced54a80-8d22-11e9-8a95-32677e4e9791.png">

The Contact Us arrow should probably align very closely with the text it's meant to be attached to, otherwise it's floating in space. 

I would explore using CSS Flexbox to change the button widths depending on the length of their contents. So if you have less content, they'll automatically align along one row; if you have more, they'll auto-break into rows.

evilluendas commented:
@sixhours I tried your suggestion and you were right, it looks better like this. It's less consistent because it looks differently depending on the language, but it looks good in any combination of label lengths.

I just commited the changes addressing the feedback 🙂

![image](https://user-images.githubusercontent.com/820374/59393442-83299d80-8d30-11e9-8ac7-2fabf26491a8.png)

![image](https://user-images.githubusercontent.com/820374/59393462-9f2d3f00-8d30-11e9-813d-598ed354771d.png)
